.github/workflows/community-pr.yml

```diff
@@ -8,7 +8,7 @@ permissions:
 jobs:
   add_community_label:
     runs-on: ubuntu-latest
-    if: ${{ github.repository == 'dotnet/roslyn-analyzers' && github.event.pull_request.user.login != 'dotnet-bot' && github.event.pull_request.user.login != 'dotnet-maestro[bot]' }}
+    if: ${{ github.repository == 'dotnet/roslyn-analyzers' && github.event.pull_request.user.login != 'dotnet-bot' && github.event.pull_request.user.login != 'dotnet-maestro[bot]' && github.event.pull_request.user.login != 'mavasani' }}
```

Comment:
I don’t think we should be special casing individual users here. What about other Roslyn team members that contribute to the repo?

Reply:
Other contributors are detected automatically because they (in general) haven't set their org visibility to private.

Reply:
Actually, it might be something else that was causing it. I think @Youssef1313 identified the root cause but it's been a while and now I forget.

Reply:
Ah, yes, it's the org visibility. Only logged-in members of the dotnet org can see that @mavasani is also a member there.